src/main/java/org/broadinstitute/hellbender/tools/exome/NormalizeSomaticReadCounts.java

```diff
@@ -189,24 +186,14 @@ private void writeTangentBetaHats(final RealMatrix tangentBetaHats, final List<S
      * @param targetFile the input target file.
      * @return never {@code null}.
      */
-    private TargetCollection<? extends BEDFeature> readTargetCollection(final File targetFile) {
+    private TargetCollection<Target> readTargetCollection(final File targetFile) {
         if (targetFile == null) {
```

Comment:
If it's `null` the code returns `null` and is not supposed to throw an exception.